src/app/calculator/compressed-air/air-leak/air-leak.service.ts

```diff
@@ -0,0 +1,627 @@
+import { Injectable } from '@angular/core';
+import { AirLeakSurveyInput, AirLeakSurveyOutput, AirLeakSurveyData, AirLeakSurveyResult, BagMethodData, CompressorElectricityData, DecibelsMethodData, OrificeMethodData, EstimateMethodData, FacilityCompressorData } from '../../../shared/models/standalone';
+import { Settings } from '../../../shared/models/settings';
+import { StandaloneService } from '../../standalone.service';
+import { OperatingHours } from '../../../shared/models/operations';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { GreaterThanValidator } from '../../../shared/validators/greater-than';
+import { ConvertAirLeakService } from './convert-air-leak.service';
+
+
+@Injectable()
+export class AirLeakService {
+
+  inputs: AirLeakSurveyInput;
+  baselineData: Array<AirLeakSurveyData>;
+  modificationData: Array<AirLeakSurveyData>;
+
+  constructor(private convertAirleakService: ConvertAirLeakService, 
+              private standaloneService: StandaloneService,
+              private formBuilder: FormBuilder) { 
+    this.inputs = this.getDefaultEmptyInputs();
+  }
+
+  getDefaultEmptyInputs(): AirLeakSurveyInput {
+    return {
+      compressedAirLeakSurveyInputVec: Array<AirLeakSurveyData>(),
+      facilityCompressorData: this.getFacilityCompressorFormReset()
+    };
+  }
+
+  getFormFromObj(inputObj: AirLeakSurveyData): FormGroup {
```

Comment:
I think maybe this should be broken into 3 forms... depends how big of an effort that will be